.gitattributes.zos

```diff
@@ -0,0 +1,23 @@
+# This .gitattributes file will cause all text files to be converted 
+# to ISO8859-1 including git's .gitattributes and .gitignore files, 
+# execpt for those selected files as defiend below, when cloned using 
+# git (from Rocket Software) on z/OS platform.
+#
+# Selected binary files will not be translated at all.
+#
+# The default for text files
+* git-encoding=iso8859-1 working-tree-encoding=iso8859-1
+
+# git's files (which MUST be ASCII)
+.gitattributes git-encoding=iso8859-1 working-tree-encoding=iso8859-1
+.gitignore git-encoding=iso8859-1 working-tree-encoding=iso8859-1
```

Comment:
Aren't these redundant, given they specify the same encodings as the default above? I suggest they be removed.

Reply:
@keithc-ca Have removed the encoding definitions that are covered by the default definition